doc/changelog/03-notations/11276-master+fix10750.rst

```diff
@@ -0,0 +1,4 @@
+- **Fixed:**
+  n:``Print Visibility`` was failing in the presence of only-printing notations
```

Comment:
The syntax ` ``xyz`` ` is for verbatim text, it gets you `xyz`; in contrast, `` :abc:`def` `` is for roles; it's like doing `\abc{def}` in LaTeX.  Maybe you want ``:cmd:`Print Visibility` `` here?

Reply:
Sorry for not being familiar with the sphinx syntax yet. Fixed.

Reply:
Quite the opposite, thanks for all you do for Coq :)